test/games/fusion/gui/test_fusion_cosmetic_patches_dialog.py

```diff
@@ -0,0 +1,66 @@
+from __future__ import annotations
+
+from PySide6 import QtCore
+
+from randovania.games.fusion.gui.dialog.cosmetic_patches_dialog import FusionCosmeticPatchesDialog
+from randovania.games.fusion.layout.fusion_cosmetic_patches import ColorSpace, FusionCosmeticPatches
+
+
+def test_suit_palette(skip_qtbot):
+    cosmetic_patches = FusionCosmeticPatches(enable_suit_palette=False)
+
+    dialog = FusionCosmeticPatchesDialog(None, cosmetic_patches)
+    skip_qtbot.addWidget(dialog)
+
+    skip_qtbot.mouseClick(dialog.suit_palette_check, QtCore.Qt.MouseButton.LeftButton)
+
+    assert dialog.cosmetic_patches == FusionCosmeticPatches(enable_suit_palette=True)
+
+
+def test_beam_palette(skip_qtbot):
+    cosmetic_patches = FusionCosmeticPatches(enable_beam_palette=False)
+
+    dialog = FusionCosmeticPatchesDialog(None, cosmetic_patches)
+    skip_qtbot.addWidget(dialog)
+
+    skip_qtbot.mouseClick(dialog.beam_palette_check, QtCore.Qt.MouseButton.LeftButton)
+
+    assert dialog.cosmetic_patches == FusionCosmeticPatches(enable_beam_palette=True)
+
+
+def test_enemy_palette(skip_qtbot):
```

Comment:
Please combine all of these test checkbox functions.
You can just feed in parameters on whats different. Should only need the field, and the checkbox.